src/test/scala/uk/gov/hmcts/ccd/corecasedata/scenarios/ESStdTest.scala

```diff
@@ -0,0 +1,208 @@
+package uk.gov.hmcts.ccd.corecasedata.scenarios
+
+import io.gatling.core.Predef._
```

Comment:
how come we are importing these ES test classes as part of this PR? These were supposed to be separately reviewed as part of another PR. If it's not a big work can we leave them out please?

Reply:
they were part of ESPT branch that I pulled upon your recommendation 🙂

Reply:
deleted ES tests, Kapil will raise a separate PR for those